app/views/puppetclasses/_form.html.erb

```diff
@@ -44,7 +44,7 @@
         </div>
         <div class="smart-var-left col-md-3">
           <ul class="nav nav-pills nav-stacked smart-var-tabs" data-tabs="pills">
-            <% @puppetclass.class_params.includes(:environments, :environment_classes, :lookup_values).each do |key| %>
+            <% @puppetclass.class_params.includes(:environments, :environment_classes, :lookup_values, :default).each do |key| %>
```

Comment:
IMHO most of the time you will want the default value with the key, maybe it should be part of the default scope for keys?